include/modmesh/python/common.hpp

```diff
@@ -135,6 +135,46 @@ namespace modmesh
 namespace python
 {
 
+struct
+MODMESH_PYTHON_WRAPPER_VISIBILITY
+ConcreteBufferNdarrayRemover : ConcreteBuffer::remover_type
+{
+
+    static bool is_same_type(ConcreteBuffer::remover_type const & other)
+    {
+        return typeid(other) == typeid(ConcreteBufferNdarrayRemover);
+    }
+
+    ConcreteBufferNdarrayRemover() = delete;
+
+    explicit ConcreteBufferNdarrayRemover(pybind11::array arr_in)
+      : ndarray(std::move(arr_in))
+    {}
+
+    // NOLINTNEXTLINE(modernize-avoid-c-arrays,cppcoreguidelines-avoid-c-arrays,readability-non-const-parameter)
+    void operator()(int8_t *) const override {}
+
+    pybind11::array ndarray;
+
+}; /* end struct ConcreteBufferNdarrayRemover */
+
+template < typename T >
+static SimpleArray<T> makeSimpleArray(pybind11::array_t<T> & ndarr)
```

Comment:
Helper function to convert from `pybind11::array_t` to `SimpleArray` while sharing the buffer.